src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -10575,24 +10574,24 @@ def test_vm_trusted_launch_os_disk_secure_upload(self):
             self.check('creationData.createOption', 'UploadPreparedSecure'),
         ])
         self.cmd('disk grant-access -n {disk} -g {rg} --access-level Write --duration-in-seconds 86400', checks=[
-            self.exists('accessSas'),
-            self.exists('securityDataAccessSas')
+            self.exists('accessSAS'),
+            self.exists('securityDataAccessSAS')
         ])
 
         self.cmd('disk create -n {disk1} -g {rg} --os-type Windows --hyper-v-generation v2 --security-type TrustedLaunch --upload-type UploadWithSecurityData --upload-size-bytes 34359738880 --sku standard_lrs', checks=[
             self.check('securityProfile.securityType', 'TrustedLaunch'),
             self.check('creationData.createOption', 'UploadPreparedSecure')
         ])
         self.cmd('disk grant-access -n {disk1} -g {rg} --access-level Write --duration-in-seconds 86400 --secure-vm-guest-state-sas', checks=[
-            self.exists('accessSas'),
-            self.exists('securityDataAccessSas')
+            self.exists('accessSAS'),
+            self.exists('securityDataAccessSAS')
```

Comment:
Why do many attribute names change? Are they breaking changes for customers?

Reply:
It is caused by deserialization of code Gen V2.
@kairu-ms could you please comment on this?

Reply:
Got it, thank you~ So they are breaking changes for customers, right? If so, do we need to add a [BREAKING CHANGE] tag to make customers aware of these breaking changes?

Reply:
Added a history note with "BREAKING CHANGE" marker:

![image](https://github.com/user-attachments/assets/19d77b70-12a0-4f7c-aa8d-8516962f967b)

Reply:
![image](https://github.com/user-attachments/assets/65309f68-6505-44c0-89ce-4beff8c0d521)
Please refine PR title as well